Description: Don't overwrite CPPFLAGS in configure.ac
Author: Tobias Frost <tobi@coldtobi.de>
Forwarded: not-needed
Last-Update: 2014-03-26
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/configure.ac
+++ b/configure.ac
@@ -9,7 +9,7 @@
 
 LT_INIT
 
-CPPFLAGS="-Wall -Wextra"
+CPPFLAGS+=" -Wall -Wextra"
 
 # we want this header to be created.
 AC_CONFIG_HEADERS(src/config.h)
